.env.example

```
@@ -177,7 +177,12 @@ REALTIME_GREETING=Hey, {bot} here. # greeting text (supports {bot})
 DEFAULT_REASONING_EFFORT=low
 DEFAULT_VERBOSITY=low
 OPENAI_REQUEST_TIMEOUT_MS=180000 # Backend OpenAI request timeout budget (3 minutes by default)
-CHAT_WORKFLOW_MODE_ID=grounded # Workflow mode id: fast, balanced, or grounded
+CHAT_WORKFLOW_MODE_ID=grounded # Workflow mode id: balanced or grounded
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHAT_WORKFLOW_MODE_ID key should go before the DATA_DIR key

(UnorderedKey)